src/adapter.js

```diff
@@ -15,9 +15,27 @@
  */
 
 (function(karma, System) {
+    System.config({ baseURL: 'base' });
 
     // Prevent immediately starting tests.
-    window.__karma__.loaded = function() {};
+    window.__karma__.loaded = function() {
```

Comment:
Can you explain how moving the following block into this `loaded` function fixes an issue with a `System.config` call? It seems like this doesn't really change when the baseUrl is configured. Just trying to understand what is going on here.